gk/main.c

```diff
@@ -417,21 +425,141 @@ setup_gk_instance(unsigned int lcore_id, struct gk_config *gk_conf)
 		goto flow_hash;
 	}
 
+	ret = snprintf(ring_name, sizeof(ring_name), "gk_ring_%d", block_idx);
+	RTE_ASSERT(ret < sizeof(ring_name));
+
+	instance->cmd_ring = (struct rte_ring *)rte_ring_create(ring_name,
+		gk_conf->flow_ht_size / 2, socket_id, RING_F_SC_DEQ);
```

Comment:
Indeed, I set it as half of the flow table size, since I didn't find a perfect way to set it. I will add a comment here.